aws/resource_aws_api_gateway_rest_api.go

```diff
@@ -76,6 +81,17 @@ func resourceAwsApiGatewayRestApiCreate(d *schema.ResourceData, meta interface{}
 
 	d.SetId(*gateway.Id)
 
+	if body, ok := d.GetOk("body"); ok {
+		_, err := conn.PutRestApi(&apigateway.PutRestApiInput{
+			RestApiId: gateway.Id,
+			Mode:      aws.String("overwrite"),
+			Body:      []byte(body.(string)),
+		})
+		if err != nil {
+			return fmt.Errorf("Error putting API Gateway specification: %s", err)
```

Comment:
It does no harm, but I think the wrapper here isn't necessary as the only place the error will escalate to is the UI, where all that matters is string-based error message user will read, not the original error type.